posts/nvtx/Makefile

```diff
@@ -33,13 +33,13 @@ BINARIES=manual_nvtx compiler_inst_nvtx
 all: $(BINARIES)
 
 inst_nvtx.o: inst_nvtx.cpp Makefile
-	g++ -fPIC -I${CUDA_ROOT}/include -c inst_nvtx.cpp
+	g++ -export-dynamic -fPIC -I${CUDA_ROOT}/include -c inst_nvtx.cpp
 
 manual_nvtx: manual_nvtx.cu Makefile
-	nvcc -DUSE_NVTX -arch=sm_20 -lnvToolsExt -o manual_nvtx manual_nvtx.cu
+	nvcc -Xcompiler -export-dynamic -DUSE_NVTX -arch=sm_20 -lnvToolsExt -o manual_nvtx manual_nvtx.cu
 
-compiler_inst_nvtx: compiler_inst_nvtx.cu Makefile inst_nvtx.o
-	nvcc -Xcompiler -fPIC -Xcompiler -finstrument-functions -arch=sm_20 inst_nvtx.o -ldl -lnvToolsExt -o compiler_inst_nvtx compiler_inst_nvtx.cu 
+compiler_inst_nvtx: compiler_inst_nvtx1.cu Makefile inst_nvtx.o
```

Comment:
You can modify this one, I believe. Either way, please make it right before
I accept.

Thanks,
Mark

On Fri, Feb 19, 2016 at 8:58 PM Vinay Deshpande <notifications@github.com>
wrote:

> In posts/nvtx/Makefile
> <https://github.com/parallel-forall/code-samples/pull/6#discussion_r53441701>
> :
>
> >
> > -compiler_inst_nvtx: compiler_inst_nvtx.cu Makefile inst_nvtx.o
> > -	nvcc -Xcompiler -fPIC -Xcompiler -finstrument-functions -arch=sm_20 inst_nvtx.o -ldl -lnvToolsExt -o compiler_inst_nvtx compiler_inst_nvtx.cu
> > +compiler_inst_nvtx: compiler_inst_nvtx1.cu Makefile inst_nvtx.o
>
> That is correct. compiler_inst_nvtx1.cu was experimental file I made.
> Should I create another pull request?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/parallel-forall/code-samples/pull/6/files#r53441701>.
>